### ISSUE TITLE
Docker client

### ISSUE THREAD

**Issue by darkdragon-001** [FoldingAtHome/fah-issues#1320]
Promote [johnktims/folding-at-home](https://github.com/johnktims/folding-at-home) as official docker client.

**Comment by cnf** [comment]
This has --gpu=false... we need one with proper GPU support...

**Comment by darkdragon-001** [comment]
You can inspire your images from [BOINC](https://hub.docker.com/r/boinc/client/).

**Comment by azak-azkaran** [comment]
This has at least cuda support:
https://github.com/coreweave/fah-docker

**Comment by jaimergp** [comment]
@amalic created this one too: https://hub.docker.com/r/amalic/nvdocker-folding-home

**Comment by beberg** [comment]
Official containers are being worked on. Coming soon.

**Comment by dmc5179** [comment]
I have containerized many version of the fahclient for both CPU and GPU support. The Dockerfiles are located here:

https://github.com/dmc5179/fah-podman

Container images are located here:

https://quay.io/organization/fah

Ansible playbook to deploy using docker or podman:

https://github.com/ajacocks/fah

The podman deploy supports GPUs through OCI hooks. docker GPU support with nvidia docker is in work.

@beberg If you're an admin for F@H let me know if you want admin access to the quay.io/fah repository. I snagged it to avoid squatting. F@H team is welcome to take it over.
